File: openapi-client-old/dart-jaguar/lib/model/send_mail_adv_from.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'send_mail_adv_from.jser.dart';

class SendMailAdvFrom {
   /* The email address */
  @Alias('email', isNullable: false,  )
  final String email;
   /* Optional contact name */
  @Alias('name', isNullable: false,  )
  final String name;
  

  SendMailAdvFrom(
      

{
    
     this.email = null,   this.name = null 
    
    }
  );

  @override
  String toString() {
    return 'SendMailAdvFrom[email=$email, name=$name, ]';
  }
}

@GenSerializer(nullableFields: true)
class SendMailAdvFromSerializer extends Serializer<SendMailAdvFrom> with _$SendMailAdvFromSerializer {

}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/SendMailAdvFrom.scala

package org.openapitools.client.model


case class SendMailAdvFrom (
    /* The email address */
    _email: String,
    /* Optional contact name */
    _name: Option[String]
)
object SendMailAdvFrom {
    def toStringBody(var_email: Object, var_name: Object) =
        s"""
        | {
        | "email":$var_email,"name":$var_name
        | }
        """.stripMargin
}